hadoop-hdds/dev-support/checkstyle/checkstyle.xml

```diff
@@ -106,45 +106,46 @@
-        <!-- <module name="Header">                            -->
-            <!-- The follow property value demonstrates the ability     -->
-            <!-- to have access to ANT properties. In this case it uses -->
-            <!-- the ${basedir} property to allow Checkstyle to be run  -->
-            <!-- from any directory within a project. See property      -->
-            <!-- expansion,                                             -->
-            <!-- http://checkstyle.sf.net/config.html#properties        -->
-            <!-- <property                                              -->
-            <!--     name="headerFile"                                  -->
-            <!--     value="${basedir}/java.header"/>                   -->
+        <!-- Checks for Headers -->
+        <!-- See http://checkstyle.sf.net/config_header.html -->
+        <!-- <module name="Header"> -->
+        <!-- The follow property value demonstrates the ability to have access to ANT properties. -->
+        <!-- In this case, it uses the ${basedir} property -->
+        <!-- to allow Checkstyle to be run from any directory within a project. -->
+        <!-- See property expansion, http://checkstyle.sf.net/config.html#properties -->
+        <!-- <property -->
+        <!--     name="headerFile" -->
+        <!--     value="${basedir}/java.header"/> -->
         <!-- </module> -->
 
         <!-- Following interprets the header file as regular expressions. -->
-        <!-- <module name="RegexpHeader"/>                                -->
-
+        <!-- <module name="RegexpHeader"/> -->
 
-        <!-- Checks for imports                              -->
+        <!-- Checks for imports -->
         <!-- See http://checkstyle.sf.net/config_import.html -->
         <module name="IllegalImport">
-          <property name="regexp" value="true"/>
-          <property name="illegalPkgs" value="^sun\..*, ^.*\.relocated\..*, ^.*\.shaded\..*"/>
-          <property name="illegalClasses" value="^org.apache.hadoop.test.(GenericTestUtils|LambdaTestUtils|TimedOutTestsListener)"/>
+            <property name="regexp" value="true"/>
+            <property name="illegalPkgs" value="^sun\..*, ^.*\.relocated\..*, ^.*\.shaded\..*"/>
+            <property name="illegalClasses"
+                      value="^org.apache.hadoop.test.(GenericTestUtils|LambdaTestUtils|TimedOutTestsListener)"/>
         </module>
         <module name="RedundantImport"/>
         <module name="UnusedImports"/>
-	<module name="AvoidStarImport"/>
-
+        <module name="AvoidStarImport"/>
+        <module name="CustomImportOrder">
+            <property name="sortImportsInGroupAlphabetically" value="true"/>
+            <property name="separateLineBetweenGroups" value="true"/>
+            <property name="customImportOrderRules" value="STATIC###THIRD_PARTY_PACKAGE"/>
+            <property name="tokens" value="IMPORT, STATIC_IMPORT, PACKAGE_DEF"/>
+        </module>
```

Comment:
Can you please include an example in the PR description for the order of imports enforced?